test/cases/ostree-simplified-installer.sh

```diff
@@ -291,6 +292,8 @@ sudo podman run -v "$PWD"/aio/:/aio:z \
 
 # TODO: tweak config aio/configs/serviceinfo_api_server.yml to test basic FDO functionalities
 #       like adding user/key/pwd, re-encryption, files, commands etc etc
+# TODO: check tpm2 config
+sudo yq -iy '.service_info.diskencryption_clevis |= [{disk_label: "/dev/vda4", reencrypt: true, binding: {pin: "tpm2", config: "{}"}}]' aio/configs/serviceinfo_api_server.yml
```

Comment:
Got error `WARNING: The scripts tomlq, xq and yq are installed in '/usr/local/bin' which is not on PATH.`. So we should use full path to call yq. like `sudo /usr/local/bin/yq -iy ...`

Reply:
fixed thanks @henrywang